pkg/apiserver/api_installer.go

```diff
@@ -487,8 +487,7 @@ func (a *APIInstaller) registerResourceHandlers(path string, storage rest.Storag
 			if hasSubresource {
 				doc = "list " + subresource + " of objects of kind " + kind
 			}
-			route := ws.GET(action.Path).To(ListResource(lister, watcher, reqScope, false, a.minRequestTimeout)).
-				Filter(m).
+			route := ws.GET(action.Path).To(metrics.InstrumentRouteFunc(action.Verb, resource, ListResource(lister, watcher, reqScope, false, a.minRequestTimeout))).
```

Comment:
I'm sorry to make this style nit, but it's pretty hard to read this right now. Can you either do it in two steps, like

```go
handler := metrics.InstrumentRouteFunc(action.Verb, resource, ListResource(lister, watcher, reqScope, false, a.minRequestTimeout))
route := ws.GET(action.Path).To(handler).
...
```

or add some line breaks like 

```
route := ws.GET(action.Path).
To(
  metrics.InstrumentRouteFunc(
    action.Verb, resource, 
    ListResource(lister, watcher, reqScope, false, a.minRequestTimeout),
  ),
).
```

Thanks :)